### PR TITLE
Add the ability to wait for new driver task will be added

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

File: test/functional/fixtures/test-run/test.js [hunk 1]
```diff
@@ -3,3 +3,9 @@ describe('Code Evaluation', function () {
         return runTests('testcafe-fixtures/evaluate-code.js', 'Evaluate');
     });
 });
+
+describe('Driver task queue', function () {
+    it('Should return real queue lengh after all commands are added', function () {
```
Comment (helen-dikareva) [inline test/functional/fixtures/test-run/test.js]: typo -> `length`

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

File: src/test-run/index.js [hunk 2]
```diff
@@ -422,8 +435,8 @@ export default class TestRun extends Session {
         var assertionTimeout = command.options.timeout === void 0 ? this.opts.assertionTimeout : command.options.timeout;
         var executor         = new AssertionExecutor(command, assertionTimeout, callsite);
 
-        executor.once('start-assertion-retries', timeout => this._enqueueCommand(new ShowAssertionRetriesStatusCommand(timeout)));
-        executor.once('end-assertion-retries', success => this._enqueueCommand(new HideAssertionRetriesStatusCommand(success)));
+        executor.once('start-assertion-retries', timeout => this.executeCommand(new ShowAssertionRetriesStatusCommand(timeout)));
```
Comment (helen-dikareva) [inline src/test-run/index.js]: We should not add ` this.debugLog.command(command);` for these commands (see [line](https://github.com/churkin/testcafe/blob/19ab20585dd46cf79456f6755e1e3b604b846a7e/src/test-run/index.js#L485))

Comment (churkin) [reply src/test-run/index.js]: Why? We have discussed with @AndreyBelym and have decided that should.

Comment (helen-dikareva) [reply src/test-run/index.js]: Yes, we should, I mix up different methods

**Pull request merged**